app/views/handlers/question.py

```diff
@@ -1,4 +1,5 @@
 from werkzeug.utils import cached_property
+from flask import url_for
```

Comment:
Imports should be alphabetized within their import block

Reply:
Thanks, fixed.